packages/apktool.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,22 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+  $toolLnk = 'apktool.lnk'
+  $category = 'Android'
+  $shimPath = 'bin\apktool.exe'
+
+  $shortcutDir = Join-Path ${Env:TOOL_LIST_DIR} $category
+  $shortcut = Join-Path $shortcutDir $toolLnk
+
+  $executablePath = Join-Path ${Env:ChocolateyInstall} $shimPath -Resolve
```

Comment:
In other packages (for example [cygwin.vm](https://github.com/mandiant/VM-Packages/tree/ee3608a53dff436bb87cac531e93d1f2d5175e40/packages/cygwin.vm)), we ensure `$toolBin` exists doing this in two steps:
```
  $toolBin = Join-Path $toolDir "bin" -Resolve
  $executablePath = Join-Path $toolBin "$toolName.exe" -Resolve
```

I am not sure if there is a reason to do this in two steps instead of how @mr-tz has done it. @MalwareMechanic do you know?

Reply:
thanks for the note, left this here as is

Reply:
I personally don't think it should matter, we were likely being over cautious before lol As long as the path is validated I don't think it matters if we do it one or two steps -- one step is shorter 😄  Now if we need to reuse the "bin" then it might make sense to have it in a separate variable, but that's not needed here.